exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java

```diff
@@ -55,44 +59,89 @@ public Viewable getStats() {
   @GET
   @Path("/stats.json")
   @Produces(MediaType.APPLICATION_JSON)
-  public List<Stat> getStatsJSON() {
-    List<Stat> stats = Lists.newLinkedList();
-    stats.add(new Stat("Number of Drill Bits", work.getContext().getBits().size()));
-    int number = 0;
-    for (CoordinationProtos.DrillbitEndpoint bit : work.getContext().getBits()) {
-      String initialized = bit.isInitialized() ? " initialized" : " not initialized";
-      stats.add(new Stat("Bit #" + number, bit.getAddress() + initialized));
-      ++number;
+  public Stats getStatsJSON() {
```

Comment:
This message gets statistics, which is great. But, it seems to combine per-cluster stats with per-drillbit info. Perhaps either have two messages (one for cluster, the other for Drillbit) or use JSON structures:

```
clusterStats: { drillbitCount: .., version: ... },
drillbitStats: { userPort: ..., ... }
```